### PR TITLE
Add tests for http handler.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Add tests for http handler.  and limit rows per page to 10000 so the client can test on the pagination easily.

1.  pagination
2. insert

## Changelog

- Not for changelog (changelog entry is not required)

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: query/src/servers/http/v1/query/result_data_manager.rs [hunk 1]
```diff
@@ -125,12 +125,20 @@ impl ResultDataManager {
...
 
         let mut end = false;
         loop {
             match ResultDataManager::receive(block_rx, tp).await {
-                Ok(block) => results.push(block_to_json(&block).unwrap()),
+                Ok(block) => {
+                    rows += block.num_rows();
+                    results.push(block_to_json(&block).unwrap());
+                    // TODO(youngsofun):  set it in post if needed
+                    if rows >= 10000 {
```
Comment (flaneur2020) [inline query/src/servers/http/v1/query/result_data_manager.rs:138]: define a constant for it?

Comment (youngsofun) [reply query/src/servers/http/v1/query/result_data_manager.rs:138]: ok

**Pull request merged**